features/step_definitions/quiz_steps.rb

```diff
@@ -0,0 +1,12 @@
+Given(/^there is a quiz called "([^"]*)"$/) do |name|
+  FactoryGirl.create(:quiz, name: name)
+end
+
+Given(/^I am on the quiz page for "([^"]*)"$/) do |quiz_name|
+  @quiz = Quiz.find_by(name: quiz_name)
```

Comment:
Oh right, I was expecting `@quiz` to get sent to the view but it didn't... I don't actually know why.